libhb/common.h

```diff
@@ -464,7 +464,7 @@ typedef enum
 
 /******************************************************************************
  * hb_job_t: settings to be filled by the UI
- * Update win/CS/HandBrake.Interop/HandBrakeInterop/HbLib/hb_job_s.cs when changing this struct
+ * Update win/CS/HandBrake.ApplicationServices/HandBrake.Interop/HandBrakeInterop/HbLib/hb_job_s.cs when changing this struct
```

Comment:
Something odd happened with this commend.  It was incorrect. HandBrake.Interop is gone. Correct path is: HandBrake.ApplicationServices\Interop\HbLib